parsers/nmparser/parse_run_details.go

```diff
@@ -71,5 +60,14 @@ func ParseRunDetails(lines []string) RunDetails {
 		covarianceTime,
 		functionEvaluations,
 		significantDigits,
+		// TODO: populate with values
```

Comment:
now that rundetails is more complex, we should not positionally match, and instead should explicitly refer to values 

```
RunDetails{
ProblemText: ""
...
}
```

Reply:
sure, I originally saw this as throw-way code to satisfy the compiler, but now realize it may stick around for a while ... change is in progress ...